linkcheck/htmlutil/linkparse.py

```diff
@@ -65,6 +65,10 @@
 # HTML anchor tags
 AnchorTags = {
     'a': ['name'],
+    'h1': ['id'],
+    'h2': ['id'],
+    'h3': ['id'],
+    'h4': ['id'],
     None: ['id'],
```

Comment:
AFAIU the `None: ['id']` means that `id` attributes on all tags (including h1-h4) be treated as anchor names, so this PR does not have any effect.

Am I mistaken?

Reply:
That is my understanding -  a bit confusing but `None` is actually acting as wild card.